supervisor/hyperpod.go

```diff
@@ -408,6 +409,20 @@ func (hp *HyperPod) createContainer(container, bundlePath, stdin, stdout, stderr
 
 	glog.Infof("createContainer() calls c.run(p)")
 	c.run(p)
+	alarm := time.After(5 * time.Second)
+	for {
+		if pid := c.ownerPod.getNsPid(); pid > 0 {
+			p.ProcId = pid
+			break
+		}
+
+		select {
+		case <-alarm:
+			glog.V(3).Infof("Timeout while waiting for container's pid")
+		default:
+			time.Sleep(100 * time.Millisecond)
+		}
+	}
```

Comment:
what's the purpose of "waiting for container's pid",
the proxy pid was created just after the hyperpod was created.
we don't need to wait it.